[2025-03-26T01:06:21.424+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: titanic_survival_classifier.model_training_task manual__2025-03-26T01:06:13.120068+00:00 [queued]>
[2025-03-26T01:06:21.443+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: titanic_survival_classifier.model_training_task manual__2025-03-26T01:06:13.120068+00:00 [queued]>
[2025-03-26T01:06:21.444+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-26T01:06:21.490+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): model_training_task> on 2025-03-26 01:06:13.120068+00:00
[2025-03-26T01:06:21.518+0000] {standard_task_runner.py:60} INFO - Started process 2410 to run task
[2025-03-26T01:06:21.523+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'titanic_survival_classifier', 'model_training_task', 'manual__2025-03-26T01:06:13.120068+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpw_arfcnb']
[2025-03-26T01:06:21.525+0000] {standard_task_runner.py:88} INFO - Job 28: Subtask model_training_task
[2025-03-26T01:06:21.568+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-26T01:06:21.639+0000] {task_command.py:423} INFO - Running <TaskInstance: titanic_survival_classifier.model_training_task manual__2025-03-26T01:06:13.120068+00:00 [running]> on host 0f3b2e2ddf0a
[2025-03-26T01:06:21.801+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='titanic_survival_classifier' AIRFLOW_CTX_TASK_ID='model_training_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-26T01:06:13.120068+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-26T01:06:13.120068+00:00'
[2025-03-26T01:06:21.819+0000] {logging_mixin.py:188} INFO - ---Training Model---
[2025-03-26T01:06:22.228+0000] {logging_mixin.py:188} INFO - Validation Accuracy: 0.8156
[2025-03-26T01:06:22.237+0000] {logging_mixin.py:188} INFO - Saved model to /opt/***/data/train/titanic_model.pkl
[2025-03-26T01:06:22.238+0000] {python.py:201} INFO - Done. Returned value was: {'validation_accuracy': 0.8156424581005587, 'model_path': '/opt/***/data/train/titanic_model.pkl'}
[2025-03-26T01:06:22.353+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=titanic_survival_classifier, task_id=model_training_task, execution_date=20250326T010613, start_date=20250326T010621, end_date=20250326T010622
[2025-03-26T01:06:22.429+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-26T01:06:22.473+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
